themes/aframe/source/css/_common.styl

```diff
@@ -188,7 +188,8 @@ h1, h2, h3, h4
     ol ol
         list-style circle
 
-    li + li
+    li + li,
+    ul ul li
```

Comment:
I assume we don't have to worry about `ol ul` or `ul ol` or `ol ol` in the future?